### PR TITLE
Update tweet link text and tweet (#1813)

### PR DESCRIPTION
**Pull request by troy**
See #1813. This:
- makes the tweet more constructive, and updates the link label to reflect that
- lets the user finish the tweet with why they want it or how they'd use it
- removes the `@TWITTERHANDLE` fake personalization, which is (a) cheesy when repeated in many tweets (looks like a spambot) and (b) a waste of characters now, since the changes above make it much more personal
- updates the translations

### CODE REVIEW

**Comment by Carlgo11** [comment]
Just out of curiosity, how are the users able to send the message to the correct place without a twitter handle?

**Comment by troy** [comment]
@Carlgo11: good point. My intent was only to modify the tweet body, but I was acting like these were @ reply tweets. I think there's a strong argument that they should be reply tweets rather than mentions like they are now. That said, I'll retain the current behavior of mentions for now. Re-adding that now.

**Comment by troy** [comment]
@Carlgo11: added at least enough to preserve the current mention behavior.

IMO, these should be @ reply tweets, like:

```
@blah Hi! ..
```

.. so they're only visible to people who care about both the sender and the recipient ([more](https://support.twitter.com/articles/14023)). That seems more fitting, since this tweet is fairly specific to the recipient - unlike the "Thanks!" tweet, it's hard to see this as general-interest. I can make that change if anyone wants.

In the interest of making this PR as narrow as possible, I didn't make that change. AFAIK this PR should be good as-is.

**Comment by Carlgo11** [comment]
as of right now we provide a template for people to easily contact a company and request 2FA.
The template can easily be changed by the user to personalize the message better.
While I like the idea of not making tweets look like they're being sent from a spambot I dislike the idea of providing an unfinished template.
That will just result in unfinished tweets.

Rather than enable the users to speak freely it gives the impression of one of those standardized tests where you're supposed to finish the sentence correctly.

I think your goal would easier be achieved by giving the users an empty template.
However our intension with the template is to provide an easy alternative to writing a new tweet to the company in question.

I have seen lots of people costomize their #SupportTwoFactorAuth tweets. (Even though it's a very small minority of the tweets)

Again, I like your idea but I think this is the wrong way to achieve it.

//Carl

**Comment by troy** [comment]
> However our intension with the template is to provide an easy alternative to writing a new tweet to the company in question.

I think that's a mediocre or even counterproductive goal. #1813 explains why, but the gist is:
- at best, it makes zero impact except making the user feel better (placates them)
- at worst, it wastes an opportunity for a useful conversation, inaccurately makes the user think they've contributed, and makes the user look bad in the process

This really comes down to whether the project is actually trying to increase the adoption of 2FA, or just encourage venting. The current tweet is venting, but it's not going to affect 2FA adoption.

The threshold for action needs to be higher than clicking "Tweet this!" because the threshold for making an impact is higher than that. If someone isn't willing to write 5-10 words, then they're tweeting to placate themselves, not to encourage 2FA adoption.

**Comment by RichJeanes** [comment]
To be fair, I don't think it's necessarily wrong to play off the expected laziness of users. We don't expect people to actually modify the tweet before sending it, so I'd rather see a complete sentence sent out than an unfinished prompt. If someone wants to edit the tweet before sending it, they're more than welcome to. 

Also, another part of it is just to generate usage of the hashtag. If the hashtag gets used enough, it has the possibility of garnering interest from Twitter visitors, not just 2fa.org visitors through trending and searches.

Also, just fyi, Twitter recently changed their policy and image links and @ mentions don't count toward the 140 character limit any more, IIRC.

**Comment by troy** [comment]
> To be fair, I don't think it's necessarily wrong to play off the expected laziness of users.

I think that comes down to the project goal. If you "don't expect people to actually modify the tweet before sending it," it doesn't encourage 2FA, it just recreates the tunnel-vision perspectives that local newspaper comment sections are known for. It's talking at someone, not talking with them.

If the goal is lots of rant-y tweets, it'll accomplish that, but that won't affect 2FA adoption. I want to see 2FA adopted, which is why I'm arguing for quality over quantity. Lots of inane tweets aren't the barrier to adoption.

If you see awareness of 2FA among less- or non-technical users as a problem, then encourage sending a single tweet in support of the concept or this site/project (without any mentions other than maybe this site). For example: "I'd love to have two factor authentication everywhere, so I #SupportTwoFactorAuth. Here's how you can too: http://.."

Basically, split out the case where talking "at" people is fine (general advocacy) from the case where talking "at" people (current tweet directed at Web apps) is painfully counterproductive.

**Comment by RichJeanes** [comment]
It may be cynical of me, but if someone isn't curious enough to get interested in 2FA from a tweet, I don't think the wording in that tweet is going to change that. 

I also find it interesting that we are discussing the practical opposite of someone else who argued that someone tweeting at just the specific sites they use was a problem (information leak). Now we face the argument that spamming tweets is a problem instead of cherry picking. I love how much variety in discussion this project generates. (I believe that point was made [here](https://github.com/2factorauth/twofactorauth/issues/1212), but the user seems to have deleted their comments for some reason...)

**Comment by troy** [comment]
So, is this dead? If so, the goal of at least the tweet feature, if not the whole project, looks much more about throwing stones than actually increasing 2FA adoption.
